site/app/templates/grading/electronic/SubmissionPanel.twig

```diff
@@ -78,7 +78,7 @@
             var directory = "";
             if (url_file.includes("submissions")) {
                 directory = "submissions";
-                url_file = getNonAnonPath(url_file, "{{anon_submitter_id}}");
+                url_file = getNonAnonPath(url_file, "{{anon_submitter_id}}", {{user_ids|json_encode|raw}});
```

Comment:
Doesn't this then break the point of using the anonymous IDs if the non-anonymized ID is printed within the page source?

Reply:
It does, this is an emergency fix since no one can view any materials. I will make an issue and address this

Reply:
This will be fixed in a future PR